### ISSUE TITLE
New Version 3.5.4 Not Signed by Application Owner

### ISSUE THREAD

**Issue by MMaloneyCRPM** [ThePacielloGroup/CCAe#352]
After installing version 3.5.3 of the CCA software from the TPGI web, the tool throws an auto update error when launched stating "New Version 3.5.4 is not signed by the application owner...."

Is there a bug in version 3.5.3?  Is version 3.5.4 available?

![Auto update error](https://github.com/user-attachments/assets/8bc76d7a-0313-4ee7-9004-996680f30ae5)

Thank you, 
mam

**Comment by ferllings** [comment]
Yes, version 3.5.4 is available. But you need to install it manually
